ports/glutin/window.rs

```diff
@@ -1026,6 +1026,17 @@ impl WindowMethods for Window {
 
     }
 
+    fn screen_size(&self, _: BrowserId) -> Size2D<u32> {
+        let (width, height) = glutin::get_primary_monitor().get_dimensions();
+        Size2D::new(width, height)
+    }
+
+    fn screen_avail_size(&self, _: BrowserId) -> Size2D<u32> {
+        // Glutin does not have APIs for getting available size, falling back to screen size.
+        let (width, height) = glutin::get_primary_monitor().get_dimensions();
+        Size2D::new(width, height)
+    }
+
```

Comment:
In the headless case, you probably want to return context.width/height for both screen_size and screen_avail_size.